label_studio/tests/sdk/fixtures.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from label_studio_sdk.client import LabelStudio
+from label_studio_sdk.data_manager import Column, Filters, Operator, Type
+
+from .common import LABEL_CONFIG_AND_TASKS
+
+
+@pytest.fixture
+def test_project_with_view(django_live_url, business_client):
```

Comment:
Makes sense to have these as fixtures now with the current way we use pytest, but looking forward to progressively replacing them with something like [Factory Boy](https://www.notion.so/humansignal/Testing-Standardization-Proposal-15a7094e54c880ed8d0ce4078c666641?pvs=4#15b7094e54c880ae98efcd905a4fd0b5) when available (cc @mcanu )